com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -820,11 +852,25 @@ protected virtual void Update()
         /// <summary>
         /// Teleports the transform to the given values without interpolating
         /// </summary>
-        public void Teleport(Vector3 newPosition, Quaternion newRotation, Vector3 newScale)
+        public void Teleport(Vector3 newPosition, Vector3 newRotationEuler, Vector3 newScale)
         {
+            if (!CanCommitToTransform)
+            {
+                Debug.LogWarning("Teleport not allowed, " + k_NoAuthorityMessage);
```

Comment:
throw exception? Or call RPC which calls this function on server?

Reply:
This would be client authority. Client would "decide" to teleport, no?

Reply:
Yeah right forget the RPC part. I still think we should throw an exception here if it's misused

Reply:
done